### ISSUE TITLE
Landing Page not rendering properly

### ISSUE THREAD

**Issue by congriholdplantains** [gophish/gophish#438]
The Landing Page was formatting properly before, but now it will render once or twice and then not render properly anymore. It seems to have to do with the external resources; css, images. I checked permissions on the repository they sit on and there is no issue. Also had no issue in the previous version and just started noticing it in the current version 0.2.

**Comment by jordan-wright** [comment]
Can you provide more information on what you mean by not rendering correctly and what you mean by "render once or twice and then not render properly anymore"? Is this in the preview?

**Comment by congriholdplantains** [comment]
I went back and loaded and older version of it and so far so good. We saw this behavior in the Landing Page template form when switching from HTML source to preview, but first noticed it when going from the link in the email to the landing page. It seemed it could not load the (external) CSS it was pointed at. I started bringing some of the CSS into the landing page itself and that's when I saw the once/twice bit. Anyway, haven't had a chance to replicate.
